### PR TITLE
Sara Kuddah

### PR DESCRIPTION
**Pull request by Sara-Kuddah**

### CODE REVIEW

**Comment by h-aljahbli-GA** [comment]
Hi Sara,

be sure to commit and push the correct files for the homework.

**Comment by Sara-Kuddah** [comment]
Done sorry for that

File: arena.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+       {this.gladiator.splice(0)}else
+
+       if(this.gladiator[0].weapon==='Club'&&this.gladiator[1].weapon==='Trident')  
+       {this.gladiator.splice(1)}else
+       if(this.gladiator[1].weapon==='Club'&&this.gladiator[0].weapon==='Trident')  
+       {this.gladiator.splice(0)}
+     console.log(this.gladiator);
+    }else {console.log('there are one or no gladiator! can not make a faght :(');}
+}
+}
```
Comment (h-aljahbli-GA) [inline arena.js:30]: dont forget to export the Arena

using 
```
export default Arena
```

File: gladiator.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
+'use strict'
+console.log('Its work !');
+class Gladiator{
+    constructor(name,weapon){
+        this.name=name;
+        this.weapon=weapon;
+    }
+}
```
Comment (h-aljahbli-GA) [inline gladiator.js:8]: you did not export gladiator either

**Comment by h-aljahbli-GA** [comment]
good job Sara!

the homework is complete, be sure to export your classes in the future.

keep it up!

**Pull request closed without merging**